src/store/reducers/commentReducers.js

```diff
@@ -0,0 +1,74 @@
+import {
+  FETCH_AN_ARTICLE,
+  FETCH_ARTICLE_COMMENTS,
+  FETCH_AN_ARTICLE_404,
+  CREATE_A_COMMENT,
+  CREATE_A_REPLY,
+  CREATE_COMMENT_SUCCESS,
+  CREATE_REPLY_SUCCESS,
+} from '../actions/actionTypes';
+
+const initialState = {
+  isFetchingArticle: true,
```

Comment:
Why is this true by default? I think it would be better to always have it as false and only let it be true if a related action is triggered. You'll realize that it might be hard to really know when the actual fetching is happening because this case will always be true in the global redux store even when no articles are being fetched ... I would suggest that you make it false by default. This way, we would know when it is triggered in the redux store. Same goes to the other state property `isFetchingComments`

Reply:
I implemented it in such a way that during fetching it displays a loading div which will be overwritten to false when `isFetchingComments` state has fetched the comments